plugins/inputs/mysql/mysql.go

```diff
@@ -94,6 +95,9 @@ const sampleConfig = `
   ## gather metrics from SHOW BINARY LOGS command output
   # gather_binary_logs = false
 
+  ## gather metrics from PERFORMANCE_SCHEMA.GLOBAL_VARIABLES
+  # gather_global_variables = false
```

Comment:
This was previously always enabled, so we should display this as true here and initialize it to true in the plugins `init()` function.